dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Admin/EditClient.cshtml

```diff
@@ -24,6 +23,14 @@
         </govuk-checkboxes-item>
     </govuk-checkboxes>
 
+    <govuk-radios asp-for="TrnRequired">
+        <govuk-radios-fieldset>
+            <govuk-radios-fieldset-legend class="govuk-fieldset__legend--m"/>
+            <govuk-radios-item value="@TrnRequirementType.Required">Yes, client requires a TRN</govuk-radios-item>
+            <govuk-radios-item value="@TrnRequirementType.Optional">No, client does not require a TRN</govuk-radios-item>
+        </govuk-radios-fieldset>
+    </govuk-radios>
```

Comment:
Using radios for edit-client and a checkbox for add-client. New clients have a yes/no choice for TRN required, so checkbox seemed appropriate. Existing apps have a legacy mode which we only want to overwrite if TRN required/optional explicitly selected

Reply:
It would be good to show the 'legacy' option here just to be safe; trying to reduce the chances of someone accidentally opting in to this new, as yet unfinished, behaviour by accident.